### PR TITLE
Save previous directory

### PR DESCRIPTION
**Pull request by robinborst95**
Last opened directory is now saved to a file, so it will be remembered when the program is closed.

### CODE REVIEW

**Comment by paulbakker65** [comment]
Nice! Looks good, did manual testing and worked great!

**Comment by Unset** [comment]
Nice! Manually tested if it saved when pressing X or cancel.

File: gui/src/main/java/net/tudelft/hi/e/gui/MainUI.java [hunk 1]
```diff
@@ -431,11 +436,44 @@ private void onRunScript() {
...
+    if (previousDirectory == null) {
+      return;
+    }
+    
+    try {
+      FileWriter writer = new FileWriter("prev_directory.txt");
+      writer.write(previousDirectory.getAbsolutePath());
+      writer.close();
+    } catch (IOException e) {
+    }
```
Comment (grootstyr) [inline gui/src/main/java/net/tudelft/hi/e/gui/MainUI.java]: Empty catch is a nogo, use a logger

**Comment by grootstyr** [comment]
Apart from two empty catches it looks fine.

**Pull request merged**